src/Illuminate/Foundation/helpers.php

```diff
@@ -288,6 +289,36 @@ function config_path($path = '')
     }
 }
 
+if (! function_exists('context')) {
+    /**
+     * Get / set the specified context value.
+     *
+     * @param  array|string|null  $key
+     * @param  mixed  $default
+     * @return mixed|\Illuminate\Log\Context\Repository
+     *
+     * @throws \InvalidArgumentException
+     */
+    function context($key, $default = null)
+    {
+        if (is_null($key)) {
+            return app(ContextRepository::class);
+        }
+
+        if (is_array($key)) {
+            if (count($key) !== 1) {
```

Comment:
@michaelnabil230 why is this limitation in place?

Reply:
@taylorotwell Sorry for this issue